app/org/zalando/markscheider/MetricsPlugin.scala

```diff
@@ -72,7 +72,7 @@ class MetricsPlugin @Inject() (
 
     if (enabled) {
       val rateUnit = configuration.getString("org.zalando.markscheider.rateUnit", validUnits).getOrElse("SECONDS")
-      val durationUnit = configuration.getString("org.zalando.markscheider.durationUnit", validUnits).getOrElse("MILLISECONDS")
+      val durationUnit = configuration.getString("org.zalando.markscheider.durationUnit", validUnits).getOrElse("SECONDS")
```

Comment:
That would possibly break clients (our, for example :)), please do not change it. You can override this value in your clients through configuration then.